com.pixelomer.ShijimaQt.yaml

```diff
@@ -0,0 +1,34 @@
+app-id: com.pixelomer.ShijimaQt
+runtime: org.kde.Platform
+runtime-version: '6.8'
+sdk: org.kde.Sdk
+command: shijima-qt
+finish-args:
+  - --device=dri
+  - --socket=x11
+  - --share=ipc
+  - --env=SHIJIMA_FLATPAK=1
+  - --socket=pulseaudio
+  - --env=QT_QPA_PLATFORM=xcb
+  # To be able to listen to and communicate with port 32456 for HTTP API and CLI
+  - --share=network
+  # Required for activating/deactivating shell extensions
+  - --talk-name=org.gnome.Shell
+  - --talk-name=org.kde.KWin
+  # Required for running gnome-extensions install
+  - --talk-name=org.freedesktop.Flatpak
+
+modules:
+  - name: shijima-qt
+    buildsystem: simple
+    build-commands:
+      - CONFIG=release make -j`nproc`
+      - install -Dm755 publish/Linux/release/shijima-qt /app/bin/shijima-qt
+      - install -Dm755 publish/Linux/release/libunarr.so.1 /app/lib/libunarr.so.1
+      - install -Dm644 com.pixelomer.ShijimaQt.desktop /app/share/applications/com.pixelomer.ShijimaQt.desktop
+      - install -Dm644 com.pixelomer.ShijimaQt.metainfo.xml /app/share/metainfo/com.pixelomer.ShijimaQt.metainfo.xml
+      - install -Dm644 com.pixelomer.ShijimaQt.png /app/share/icons/hicolor/512x512/apps/com.pixelomer.ShijimaQt.png
```

Comment:
why aren't these installed by make?

Reply:
So far the project has mostly been distributed/tested as an AppImage and as a sole binary so `make install` hasn't been necessary. Should I upstream this as well?

Reply:
this should be inside app image too, a desktop file distributed without icon desktop file is a bug.

Reply:
The app images do contain the desktop file and icon, but the app directory is generated by linuxdeploy in that case.